### ISSUE TITLE
2nd Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#3]
### 진행 전
- Franklin Clinton, Michael De Santa, Trevor Philips 총 3종류로 100x100x3 사이즈의 각 600장 사진을 구비함(Train: 360개, Test: 240개)
- 기존 MATLAB으로 딥러닝을 하는게 무의미하다고 판단, Jupyter Notebook과 Tensorflow를 통해 딥러닝으로 노선 변경
- CNN 구조는 AlexNet에서 임의로 구현한 NNN(NaminNeuralNetwork)으로 변경
![NaminNeuralNetwork](https://user-images.githubusercontent.com/47406388/68735131-10df2100-0620-11ea-8d50-23be131ab2ef.jpg)
### 진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현 (Conv1-Relu1-Pool1-Conv2-Relu2-Pool2-Affine)
- Softmax로 확률 구현
-Adam으로 Optimizer 구현
-Batchsize는 1080, Epoch은 1000으로 훈련
-훈련 파일을 ckpt 확장자로 저장 후 시험
### 진행 후
- 훈련 도중 오차율 기울기 손실 (344회 훈련지점)
[NNNTraining.txt](https://github.com/roqhdehd502/Franklin-Detecter/files/3839527/NNNTraining.txt)
- 시험 결과 Acc:1.0으로 나왔는데 Overfitting 혹은 오류로 보임
```
INFO:tensorflow:Restoring parameters from logs\model.ckpt-1000
1.0
```
- 추후 기존 CNN 구조에서 Batch Normalization등 Overfitting 및 기울기 손실이 일어나지 않도록 조정할 것
